com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Camera/HDCameraUI.Rendering.Drawers.cs

```diff
@@ -150,9 +154,14 @@ static void Drawer_Draw_DLSS_Section(SerializedHDCamera p, Editor owner)
                         featureDetected ? MessageType.Info : MessageType.Warning);
                 }
             }
-
 #endif
+            static void Drawer_Draw_FSR_Section(SerializedHDCamera p, Editor owner)
+            {
+                if (!p.allowDynamicResolution.boolValue)
+                    return;
 
+                EditorGUILayout.PropertyField(p.fsrSharpness, HDRenderPipelineUI.Styles.fsrSharpnessText);
```

Comment:
We should add a property as well in the quality assert settings, and then an override in the camera. 
If you want to stick to a property in the camera, i'd say you either hide it if the quality asset has a different FSR method, or put a note in the camera that says its currently disabled.